test/helpers/kubectl.go

```diff
@@ -4394,6 +4394,16 @@ func (kub *Kubectl) CleanupCiliumComponents() {
 			"service":            "cilium-agent hubble-metrics hubble-relay",
 			"secret":             "hubble-relay-client-certs hubble-server-certs",
 		}
+
+		crdsToDelete = []string{
+			"ciliumclusterwidenetworkpolicies.cilium.io",
+			"ciliumendpoints.cilium.io",
+			"ciliumexternalworkloads.cilium.io",
+			"ciliumidentities.cilium.io",
+			"ciliumlocalredirectpolicies.cilium.io",
+			"ciliumnetworkpolicies.cilium.io",
+			"ciliumnodes.cilium.io",
+		}
```

Comment:
You could consider referencing: https://github.com/cilium/cilium/blob/d390b97ba730585c285620f70d255a70e49098ce/pkg/k8s/synced/crd.go#L61

...might be worth importing that package, or not. Just pointing it out to raise awareness. 

Edit: realized that those all start with the prefix "crd:", which makes them immediately not useful as is to replace `crdsToDelete`, but I'll leave this comment for posterity.

Reply:
Thanks! I was grepping for such a list. Splitting the `crd` part seems easy enough though, I'll switch over to using `AllCRDResourceNames` to avoid increasing the maintenance burden.